vhdbuilder/packer/install-dependencies.sh

```diff
@@ -74,15 +74,17 @@ cat << EOF >> ${VHD_LOGS_FILEPATH}
 EOF
 
 VNET_CNI_VERSIONS="
+1.1.2
+1.1.0
 1.0.33
-1.0.29
 "
 for VNET_CNI_VERSION in $VNET_CNI_VERSIONS; do
-    VNET_CNI_PLUGINS_URL="https://acs-mirror.azureedge.net/cni/azure-vnet-cni-linux-amd64-v${VNET_CNI_VERSION}.tgz"
+    VNET_CNI_PLUGINS_URL="https://acs-mirror.azureedge.net/azure-cni/v${VNET_CNI_VERSION}/binaries/azure-vnet-cni-linux-amd64-v${VNET_CNI_VERSION}.tgz"
```

Comment:
Too bad I didn't catch this issue. This requires a change in downloadAzureCNI() method, otherwise the downloaded file name wouldn't be the same as CSE expected.
tony@test05211804:/opt/cni/downloads$ ls -ltr
total 100988
-rw-r--r-- 1 root root 22304370 May 21 00:54 v1.1.2
-rw-r--r-- 1 root root 22301518 May 21 00:54 v1.1.0
-rw-r--r-- 1 root root  7464179 May 21 00:54 v1.0.33
-rw-r--r-- 1 root root 17109530 May 21 00:54 cni-plugins-amd64-v0.7.6.tgz
-rw-r--r-- 1 root root 17109361 May 21 00:55 cni-plugins-amd64-v0.7.5.tgz
-rw-r--r-- 1 root root 17108856 May 21 00:55 cni-plugins-amd64-v0.7.1.tgz


vs

tony@test0512:~$ cd /opt/cni/downloads/; ls -ltrh
total 64M
-rw-r--r-- 1 root root 7.2M May 13 18:27 azure-vnet-cni-linux-amd64-v1.0.33.tgz
-rw-r--r-- 1 root root 7.2M May 13 18:27 azure-vnet-cni-linux-amd64-v1.0.29.tgz
-rw-r--r-- 1 root root  17M May 13 18:27 cni-plugins-amd64-v0.7.6.tgz
-rw-r--r-- 1 root root  17M May 13 18:27 cni-plugins-amd64-v0.7.5.tgz
-rw-r--r-- 1 root root  17M May 13 18:27 cni-plugins-amd64-v0.7.1.tgz

We will need to build a new VHD.